src/components/CardSection.vue

```diff
@@ -0,0 +1,113 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+import Card from './Card.vue';
+
+export default defineComponent({
+	name: 'CardSection',
+	components: {
+		Card
+	},
+
+	data() {
+		return {
+			id: 1,
+		}
+	},
+
+	props: {
+		sectionTitle: {
+			type: String,
+			default: '',
+		},
+	},
+
+	computed: {
+		sectionId(): string {
+			if (this.sectionTitle === '') {
+				return `section-${this.id++}`;
+			}
+			return this.sectionTitle.toLowerCase().replace(/\s/g, '-');
+		}
+	}
+});
+</script>
+
+<template>
+	<section :id=sectionId>
+		<h2 v-if="sectionTitle !== ''" class="section-header">{{ sectionTitle }}</h2>
+		<h2 v-else class="screenreader-only">Links</h2>
```

Comment:
**Suggestion:**
```suggestion
		<h2 :class="sectionTitle ? 'section-header' : 'screenreader-only'>
		    {{ sectionTitle || 'Links' }}
		</h2>
```

Reply:
Ooo... nice

Reply:
So, actually.. this makes TS unhappy. I think because of the `if (sectionTitle)`. Not going to worry about changing this for now.